api/rs/slint/Cargo.toml

```diff
@@ -91,12 +91,10 @@ renderer-skia = ["i-slint-backend-selector/renderer-skia", "std"]
 ## The Skia based rendering engine that will always use OpenGL.
 renderer-skia-opengl = ["i-slint-backend-selector/renderer-skia-opengl", "std"]
 
-## This feature makes the `slint::platform::swrenderer` module available in the public API.
 ## Use this to render with Slint for example on MCUs, when you provide you own `slint::platform::Platform`.
 ## Alternatively, enabling this feature alongside `eventloop-winit` enables selecting the software
 ## renderer at run-time when using winit, via the `SLINT_BACKEND=software` environment variable.
 renderer-software = [
```

Comment:
you can get rid of this feature completely here, and in the selector, and everywhere

Reply:
Ah, maybe not, because of the optional dependency to femtovg in winit.

But the documentation need to say that this is just about winit then..

Reply:
So how about this as a doc-string then:

> ## Enabling this feature alongside `eventloop-winit` enables selecting the software renderer at run-time when running winit, via the `SLINT_BACKEND=software` environment variable.

I thought we wanted to keep the software renderer off by default at this time, so I thought leaving this and requiring an env var to be set makes sense:-) But then I only read this text and did not investigate any further how that actually works behind the curtains.

Reply:
yeah, i guess that's right

Reply:
New version has that now.